### PR TITLE
Use latest release.yaml

### PR DESCRIPTION
**Pull request by chmouel**
No need for auto-dection magic to detect triggers/pipeline releases

We can just use the official latest/release.yaml for the projects

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

```release-note
NONE
```

### CODE REVIEW

File: test/e2e-common.sh [hunk 1]
```diff
@@ -138,8 +138,7 @@ function install_pipeline_crd() {
         latestreleaseyaml=https://storage.googleapis.com/tekton-releases-nightly/pipeline/latest/release.yaml
 
     # If for whatever reason the nightly release wasnt there (nightly ci failure?), try the released version
-    [[ -z ${latestreleaseyaml} ]] && \
-        latestreleaseyaml=$(curl -s https://api.github.com/repos/tektoncd/pipeline/releases|python -c "import sys, json;x=json.load(sys.stdin);ass=x[0]['assets'];print([ x['browser_download_url'] for x in ass if x['name'] == 'release.yaml'][0])")
+    [[ -n ${latestreleaseyaml} ]] || latestreleaseyaml="https://storage.googleapis.com/tekton-releases/latest/release.yaml"
```
Comment (danielhelfand) [inline test/e2e-common.sh:141]: Should this be the following?

```suggestion
    [[ -n ${latestreleaseyaml} ]] || latestreleaseyaml="https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml"
```

`https://storage.googleapis.com/tekton-releases/latest/release.yaml` installs version v0.10.1.

Comment (chmouel) [reply test/e2e-common.sh:141]: you are absoluty right, thanks for spotting this

**Review by piyush-garg — APPROVED** [review]
/lgtm

**Pull request merged**